website/src/pages/collections/collection.page.tsx

```diff
@@ -36,7 +36,17 @@ const CollectionPage = () => {
           <ul>
             {documents.map((document) => (
               <li key={document.slug}>
-                <Link href={documentRoute(document.slug)}>
+                <Link
+                  href={
+                    // If this document exists within a collection's chapter, then redirect to that chapter page.
+                    document.chapterPath && document.chapterPath.length > 1
+                      ? chapterRoute(
+                          document.chapterPath[0]!,
+                          document.chapterPath[document.chapterPath.length - 1]!
+                        )
+                      : documentRoute(document.slug)
+                  }
```

Comment:
This might be better as a new function alongside `chapterRoute` and `documentRoute`